### DISCUSSION TITLE
Git error when trying to add changes

### DISCUSSION THREAD

**Discussion by christianeri** [DevSkillsHQ/discusssions#59] [Help]
I created an implementation branch and used Visual Studio to clone the GitHub repository down to my local machine running Windows 11.
As soon as I try to use the "git add ." command I get errors stating: "error: unable to index file" and "fatal: adding files failed". 

![Error-unable-to-index-file](https://github.com/DevSkillsHQ/discusssions/assets/83070959/fd5a59cb-4dcf-4bd2-8ea2-743325e629f8)

It seems like this error is related to my local Visual Studio installation but I have never encountered it before I the coding test. None of the suggestions for solutions or work arounds I found online at the time solved the issue. 

I tried to upload my solution to the remote implementation branch of the repository using GitHubs drag-and-drop function but I was denied as my solution contained more than 100 files which apparently is the limit.

**Comment by fenske** [comment]
Hi! I'm Anton from Alva. Happy to help.

Seems like you user in the Powershell doesn't have the necessary permissions to access the referenced file.
 Try `chmod 777 {filename}` and then git add . again. Does it work?

If not, feel free to attach you submission as a zip file here, and I'll submit it on your behalf.

**Comment by christianeri** [comment]
I get "chmod : The term 'chmod' is not recognized as the name of a cmdlet, function..." I'm on a Windows 11 machine.
Thanks, attached below:
[alva-xlent.zip](https://github.com/DevSkillsHQ/discusssions/files/13717022/alva-xlent.zip)

**Reply by fenske** [reply]
Sorry for a bit of a slow reply but I've now submitted your code.

Note: some API tests are failing due to error HTTP 500. You could reproduce it by spinning up a Codespace as per README instructions. 

I think the easiest would be to turn on the debug log and see what is failing exactly (not a .NET developer myself so didn't know how to do it).

BR,
Anton.
